include/megaapi_impl.h

```diff
@@ -172,11 +172,20 @@ class MegaSizeProcessor : public MegaTreeProcessor
         long long getTotalBytes();
 };
 
-class MegaFolderUploadController : public MegaRequestListener, public MegaTransferListener
+class MegaRecursiveTransferOpertaion
```

Comment:
This interface could be renamed to a more generic MegaRecursiveOperation (btw, there's a typo now: `Opertaion`)

Reply:
thanks, renamed (with typo corrected :-) ) cheers